core-skills/frontend-web-development/README.md

```diff
@@ -0,0 +1,45 @@
+# Frontend web developement [HTML/CSS]
+
+## Aim
+This course is intended to guide learning to a level where you can write and contribute semanitic, and maintainable HTML and CSS to a project.
+
+The course guides you through a typical "first page" build of any green field project. This encompasses selecting/creating a responsive CSS grid, building core shared components for your new site, and finally delivering an accessible, SEO optimised, lightning fast loading product detail page.
+
+## Assessment Style
+This core skill assessment is split into two parts. Firstly, you'll take a design from PSD to HTML/CSS during Learn Tech. This is assessed over three levels; Rhino, Narwhal, and Unicorn. Secondly, before completing each level as HTML/CSS is quite subjective you'll be asked to talk about the decisions you made whilst building out the deliverables for each level.
+
+In addition to the deliverables, at each level you are expected to demonstrate:
+
+- Pixel Perfect HTML/CSS (Your preferred browser)
+- Responsive Design
+- Cross Browser testing (Chrome/Firefox/Edge/Safari)
+- Semantic HTML (i.e. no [Tag Soup](https://en.wikipedia.org/wiki/Tag_soup) )
+- Logical CSS naming
+
+It is up to the assessor to determine whether or not the candidate has adequetly achieved the above in addition to the deliverables. 
+
+As this isn't an assessment about scaffolding an application, and gathering resources we have built the (base application)[https://github.com/madetech/core-skill-frontend]. At this time this repo is private to Made Tech members, we're working to open source this too.
+
+## Core Concepts
+Each of the sections below contain reading material and links to related resources and tools to help you on your way. Some of them wont be assessed but are good things to know about.
+
+- [Grids](#)
```

Comment:
Do we have content for these areas yet or empty?

Reply:
We have them in a doc, just need to format